fixtures/small/heredocs_expected.rb

```diff
@@ -69,6 +69,19 @@ class Foo
   rubocop.smash(bad_thing)
 end
 
+this_one_is
+  .in_a_call_chain {
+    # some stuff
+  }
+  .each do
+    <<~MYHEREDOC
```

Comment:
crimes

Reply:
on the downside, i'm currently still debugging even more crimes